src/main/resources/marc4bibframeAutority.yml

```diff
@@ -0,0 +1,65 @@
+authorityFieldRules:
+  '100':
+    - types: CONCEPT, PERSON
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '!3'
+      edges:
+        - types: PERSON
+          predicate: FOCUS
+          subfields:
+            a: NAME
+            b: NUMERATION
+            c: TITLES
+            d: DATE
+            j: ATTRIBUTION
+            q: NAME_ALTERNATIVE
+    - types: CONCEPT, FAMILY
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '3'
+      edges:
+        - types: FAMILY
+          predicate: FOCUS
+          subfields:
+            a: NAME
+            b: NUMERATION
+            c: TITLES
+            d: DATE
+            j: ATTRIBUTION
+            q: NAME_ALTERNATIVE
+
+
+sharedAuthorityRules:
+  100_mappings:
+    predicate: CLASSIFICATION
+    subfields:
+      a: NAME
+      b: NUMERATION
+      c: TITLES
+      d: DATE
+      j: ATTRIBUTION
+      q: NAME_ALTERNATIVE
+      v: FORM_SUBDIVISION
+      x: GENERAL_SUBDIVISION
+      y: CHRONOLOGICAL_SUBDIVISION
+      z: GEOGRAPHIC_SUBDIVISION
+    constants:
+      RESOURCE_PREFERRED: true
+    edges:
+      - types: FORM
+        predicate: SUB_FOCUS
+        subfields:
```

Comment:
Let us add `multiply: true` here (and in the other SUB_FOCUS edges also) so that multiple outgoing edges will be created if $v subfield is repeating.